data/configuration.xml

```diff
@@ -1249,7 +1249,7 @@
                                         binding</ulink>. Giving users full access to all Java
                                     classes should be considered a security risk and the feature is
                                     thus disabled by default. If you enable it, you should think
-                                    about configuring <ulink url="xacml.xml">XACML</ulink> to
+                                    about configuring <ulink url="security.xml#ACLs">ACL</ulink> to
```

Comment:
Duncan, XACML is very different from ACL. We simply remove XACML and no longer support it, so I think it might be better just to delete this bit.